src/plugins/ui_core_views/header_sizes_ui.ts

```diff
@@ -21,7 +21,7 @@ interface CellWithSize {
   size: Pixel;
 }
 
-export class HeaderSizeUIPlugin extends UIPlugin<HeaderSizeState> implements HeaderSizeState {
```

Comment:
note that this plugin depends on DOM api to create a canvas element. We won't be able to move it to a Worker as it is now, we'll have to refactor a bit more here.
`private ctx = document.createElement("canvas").getContext("2d")`

Reply:
I didn't know OffscreenCanvas were a thing